src/cellink/widgets/codeeditor.h

```diff
@@ -109,12 +110,14 @@ private slots:
 private:
     QString textUnderCursor() const;
     void paintLineNumbers(QPainter* painter, const QRect& rect);
+    void updateHighlightLines();
 
     friend class LineNumberBar;
 
     LineNumberBar m_lineNumberBar{this};
     QCompleter* m_completer = nullptr;
     qreal m_highlightLineColorAlpha = 0.2;
+    QSet<int> m_selectedLines{};
```

Comment:
time to pimpl this, as qset is a big one, IMO

Reply:
I removed the QSet again, but probably worth to PIMPL this either way soon